### DISCUSSION TITLE
Welcome to JobSet Discussions!

### DISCUSSION THREAD

**Discussion by DevSwayam** [TechieeGeeeks/JobSet#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by raghchawla131** [comment]
Hello! I'm Raghav, and I'm enthusiastic about contributing to your project. My main motivation is to gain valuable experience in open source development.
After cloning the project locally, i am facing this error. Please help me to run the project locally
<img width="819" alt="image" src="https://github.com/TechieeGeeeks/JobSet/assets/112266194/79518b58-f98d-4c5d-9bc5-e1259b55561c">

**Reply by DevSwayam** [reply]
I like your efforts to start discussion. Thank you so much for starting out discussion.

First of all your using npm install in base directory
there are two folders
![image](https://github.com/TechieeGeeeks/JobSet/assets/99035115/c247773a-f960-4344-89cd-396ffed8b39c)

You have to cd Frontend and then use command npm install and then cd backend and then do npm install.
Sorry i guess i didnt mention this in readme wil do it.
